torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1444,6 +1444,8 @@ def method_tests():
         ('__getitem__', torch.randn(S, S, S), (dont_convert([[0, 2, 3], [1, 3, 3],
                                                              torch.LongTensor([0, 0, 2])]),), 'adv_index_var'),
         ('to_sparse', (S, S), (), '', (), (), [], lambda x: x.to_dense()),
+        ('linalg.tensorsolve', (2, S, M), ((2, S),),
```

Comment:
Most of our internal build issues have been resolved but there's still an internal problem resolving the references to the tests generated here. Would you just revert the changes to common_methos_invocations.py for a moment? We should move these tests to use OpInfos soon, anyway. The temporary gap in test coverage is OK. 

Sorry to keep bringing these internal tooling issues to you, @IvanYashchuk.

Reply:
Okay, I removed it.